### PR TITLE
chore(release): 0.48.0

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

File: tests/specs/statements/sheBang/SheBang_All.txt [hunk 1]
```diff
@@ -53,7 +53,6 @@
 
 [expect]
 #!/foo/bar
-
```
Comment (dsherret) [inline tests/specs/statements/sheBang/SheBang_All.txt:56]: Hmmm

**Comment by dsherret** [comment]
Don't publish yet. I'm going to look into that test.

File: tests/specs/statements/sheBang/SheBang_All.txt (hunk 1 above)
Comment (dsherret) [reply tests/specs/statements/sheBang/SheBang_All.txt:56]: Damn. This is an swc bug based on my fix.

Comment (bartlomieju) [reply tests/specs/statements/sheBang/SheBang_All.txt:56]: Yikes, sorry about that 😬

**Comment by joscha** [comment]
@dsherret @bartlomieju This is the only change that I could find which seems somewhat related - when forking this repository the build that passed in this repo seem to fail in the fork with some diff in `SheBang_All.txt`, See https://github.com/joscha/dprint-plugin-typescript/pull/1/checks?check_run_id=3063996473 which is a fork of HEAD at 32f8148 plus an unrelated file (see https://github.com/joscha/dprint-plugin-typescript/pull/1). Any ideas on why this would suddenly start failing? Is it possible that we depend on an external dependency of which we don't control the version exactly (or the transitive closure of all dependencies of it)? If yes, that would mean that a rebuild of 32f8148 in this repo (https://github.com/dprint/dprint-plugin-typescript/runs/2992035371) should fail as well now.

//cc @declanvong

**Comment by dsherret** [comment]
@joscha I've fixed it here https://github.com/dprint/dprint-plugin-typescript/commit/8ebf7fc96170d35ed621d8457bd1f21f252eefe0

It's because I did a workaround for an swc bug, then fixed the bug in swc which must have landed in a patch release. Should be good now.

**Comment by joscha** [comment]
Thank you @dsherret - just so I understand the fix in 8ebf7fc - it seems swc's version was fixed to a specific patch release, how come a new build at a later point in time would have picked a new one up? Is there some semver range defined anywhere?

**Comment by dsherret** [comment]
@joscha there’s no lock file (from my understanding, similarly to if someone depended on this crate) so cargo will pick up the latest patch builds (in this case a patch build of swc_ecma_parser transitively through swc_ecmascript).

I think the way semver works with cargo is even if you specify a patch version, it will still pull in the latest patch (we actually found when publishing it seems to do this even with a lock file). You need to add an equals sign before it to be pinned to that version and I don’t think that would help in this case because I don’t think that pins dependencies of a dependency.

**Comment by joscha** [comment]
got it, thank you for the explanation, appreciate it!

**Comment by joscha** [comment]
just googled a bit more and wondered if we should introduce a Cargo.lock (https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html) it seems that pins dependencies and their transitive dependencies.

**Pull request merged**